examples/dwarfdump.rs

```diff
@@ -427,6 +433,24 @@ fn main() {
     }
     if matches.opt_present("i") {
         flags.info = true;
+        if let Some(offset) = matches.opt_str("i") {
+            let offset = match offset
+                .get(2..)
```

Comment:
Would be nice if the `0x` was optional.

Reply:
My primary concern was that we were skipping the first two characters without checking what they were. `llvm-dwarfdump` treats it as decimal if `0x` is missing, so I guess we need to match that, or give an error.